DEPLOY_LINUX.md

```diff
@@ -0,0 +1,87 @@
+
+## Deploy the Console on Linux
```

Comment:
i'm not a fan of anything 'os specific' for the ZAC. there's nothing specific to linux here other than the sample commands and i'm fine with using bash as example since that's availble 'mostly' in all the os's